### ISSUE TITLE
Document clearing data from coordinator

### ISSUE THREAD

**Issue by colton-citus** [citusdata/citus_docs#869]
If user takes the following steps, data will be left on coordinator:

1. Create table
2. Load data
3. Distribute table

Fix:

```
SET citus.enable_ddl_propagation TO false;
TRUNCATE TABLE target_table;
SET citus.enable_ddl_propagation TO true;
```

**Comment by jonels-msft** [comment]
The broader use case is when someone happened to load the data onto the coordinator prior to distributing it, and now wants to reclaim space on the coordinator?

**Comment by colton-citus** [comment]
Yes, that is correct. I'd also suggest adding a warning and clearing steps so this doesn't surprise customers too often.
